protopipe/aux/example_config_files/analysis.yaml

```diff
@@ -3,6 +3,8 @@
 General:
  config_name: 'example'
  site: 'south'  # 'north' or 'south'
+ zenith_distance: '20deg'
+ pointing: 'south'
```

Comment:
These variables are not read by the DL3 script - did you meant to put them in `performance.yaml` perhaps?

Reply:
Also instead of pointing (which is the term for the pair of coordinates) use Az where the values are 0, 180 or average